packages/docs/src/gatsby-plugin-theme-ui/index.js

```diff
@@ -127,6 +129,7 @@ export default {
         color: 'inherit',
       },
     },
+    prism,
```

Comment:
This looks odd to me, I'd rather not mix in component-specific props with the `theme.styles` object